website/yarn.lock

```diff
@@ -9,35 +9,25 @@
   dependencies:
     "@babel/highlight" "^7.0.0"
 
-"@babel/code-frame@^7.10.4":
```

Comment:
It's something wrong with the Yarn lockfile, because dependencies are downgraded. 

Is the branch up to date with the `master`?

Reply:
We have not intensionally made any changes to the dependencies, we just extended @motiz88 prior-work within the scripts directory and just manually changed the version of `@motiz88/react-native-docgen` from 0.0.2 to 0.0.3 as we were using the module with yarn link from a local clone of `@motiz88/react-native-docgen` as the latest version was not published before.

Reply:
Also, before creating the PR our branch was up to date. Still, I will verify the branch even with `master`

Reply:
After a check, as I suspected, the reason for the lock file problem is outdated PR branch. 

I'm going to fetch the changes locally and push only lock file diff (to avoid rebasing and potential issues in this PR), but next time please ensure that PR branch is even with `master`.

Reply:
Sure! @Simek thanks 🚀